app/javascript/components/recipes/index/filters.vue

```diff
@@ -16,12 +16,36 @@
     </button>
     <!-- Filters -->
     <div
-      v-for="filter in filterOptions"
-      :key="filter"
-      class="flex flex-row inline-block items-center justify-center px-3 py-3 static w-auto h-8
-      rounded-full flex-none flex-grow-0 mx-2.5 bg-yellow-500 text-white"
+      v-if="filters.price.min !== '' || filters.price.max !== ''"
```

Comment:
acá podría quedar más simple así: `v-if="filters.price.min || filters.price.max"` porque los [strings vacíos en js](https://www.nfriedly.com/techblog/2009/07/advanced-javascript-operators-and-truthy-falsy/) son `falsey` y se evalúan a `false` .

Reply:
Perfect! No sabia eso! Ya lo cambié. Gracias Andrés